In [4]:
import pdfplumber as plu
pdf=plu.open("it_resume.pdf")
page=pdf.pages[0]
text = page.extract_text()

In [5]:
print(text)

Tushar Wansinge
VJTI, Mumbai, Maharashtra
(cid:131) 9168787276 # tusharwansinge45@gmail.com #tvwansinge 21@ee.vjti.ac.in
b
Education
VJTI Dec 2021 – current
Bachelor of Technology in Electrical Engineering. (CGPA - 8.18) Mumbai
Hislop Junior College Nagpur 2019 – 2021
Maharashtra State Board,Pune (HSC Percentage - 93, MHT-CET Percentile - 96.389) Nagpur, Maharashtra
Chanda Public School,Chandrapur 2019
Central Board of Secondary Education (SSC Percentage - 89.8) Chandrapur, Maharashtra
Relevant Coursework
• Electrical Machines • Power Systems • Power Electronics
• Electrical Networks • Analog and Digital Circuits • Project Management
Personal Projects
Power BI Dashboards
• Built an interactive Logistics dashboard which shows various insights of parameters related to Key Performance Indicators.
• Performed data cleaning in Excel on given data and turned into an interactive dashboard in Power BI.
Electrical Vehicle design using MATLAB Software
• Simulated a design of Electric Vehicle on 

In [6]:
import pdfplumber

with pdfplumber.open("Resume 1.pdf") as pdf:
    page = pdf.pages[0]
    text = page.extract_text()

# Clean the text
text = text.replace('\n', ' ')         # remove line breaks

print(text)


YASH MHATRE +91-7020257087 Roll No.: 241030901 mailto:ymmhatre_b23@ee.vjti.ac.in ELECTRICAL ENGINEERING Veermata Jijabai Technological Institute, Mumbai ï Yash Mhatre Education Veermata Jijabai Technological Institute (VJTI) • Bachelor of Technology in Electrical Engineering AUG.2025-CURRENT Veermata Jijabai Technological Institute (VJTI) CGPA: 8.61 • June 2021-May 2024 Diploma in Electrical Engineering Percentage:96.89 • M.G.V HIGHSCHOOL May 2020 SSC,Board Percentage: 91.80 Experience Brihanmumbai Electricity Supply and Transport Undertaking • (JUNE23-JULY23) Industrial Trainee Mumbai – learn about maintenance of underground transmission lines, & fault detection. – Gained knowledge of preventive maintenance and power distribution systems. The Tata Power Company Limited (JUNE25-JULY25) • INTERNSHIP Mumbai – Gained training of the cable department and different types of HT/EHV cables – Learned cable maintenance practices, fault locating, and jointing techniques. Personal Projects Hybrid

YASH MHATRE +91-7020257087 Roll No.: 241030901 mailto:ymmhatre_b23@ee.vjti.ac.in ELECTRICAL ENGINEERING Veermata Jijabai Technological Institute, Mumbai ï Yash Mhatre Education Veermata Jijabai Technological Institute (VJTI) • Bachelor of Technology in Electrical Engineering AUG.2025-CURRENT Veermata Jijabai Technological Institute (VJTI) CGPA: 8.61 • June 2021-May 2024 Diploma in Electrical Engineering Percentage:96.89 • M.G.V HIGHSCHOOL May 2020 SSC,Board Percentage: 91.80 Experience Brihanmumbai Electricity Supply and Transport Undertaking • (JUNE23-JULY23) Industrial Trainee Mumbai – learn about maintenance of underground transmission lines, & fault detection. – Gained knowledge of preventive maintenance and power distribution systems. The Tata Power Company Limited (JUNE25-JULY25) • INTERNSHIP Mumbai – Gained training of the cable department and different types of HT/EHV cables – Learned cable maintenance practices, fault locating, and jointing techniques. Personal Projects Hybrid Power Plant Analysis • – Analysed a village site for a solar-wind hybrid power plant. – Evaluated load requirements using electricity bills. – Calculated solar panel and wind turbine needs. – Tools & technologies used: MULTISIM IOT Based Electrical Machine Protection System • - This project is designed to continuously monitor the health of a three-phase induction motor using current, voltage, and temperature sensors connected to an ESP32 development board. – Tools & technologies used: Arduino IDE, Technical Skills - Languages: Basic C, Python, Basic MATLAB,Python(Libraries) - Software Tools: Multisim ,MATLAB ,Arduino IDE Positions of Responsibility Head Of Operation (Electrical • Engineering Student Association) Achievements VJTI - ALCHEMY 2025 Project competition March-2025 first price.

In [7]:
import pdfplumber as plu

pdf=plu.open("Resume 1.pdf")
page=pdf.pages[0]

text=page.extract_text()

para=text.replace("\n"," ")

In [8]:
print(para)

YASH MHATRE +91-7020257087 Roll No.: 241030901 mailto:ymmhatre_b23@ee.vjti.ac.in ELECTRICAL ENGINEERING Veermata Jijabai Technological Institute, Mumbai ï Yash Mhatre Education Veermata Jijabai Technological Institute (VJTI) • Bachelor of Technology in Electrical Engineering AUG.2025-CURRENT Veermata Jijabai Technological Institute (VJTI) CGPA: 8.61 • June 2021-May 2024 Diploma in Electrical Engineering Percentage:96.89 • M.G.V HIGHSCHOOL May 2020 SSC,Board Percentage: 91.80 Experience Brihanmumbai Electricity Supply and Transport Undertaking • (JUNE23-JULY23) Industrial Trainee Mumbai – learn about maintenance of underground transmission lines, & fault detection. – Gained knowledge of preventive maintenance and power distribution systems. The Tata Power Company Limited (JUNE25-JULY25) • INTERNSHIP Mumbai – Gained training of the cable department and different types of HT/EHV cables – Learned cable maintenance practices, fault locating, and jointing techniques. Personal Projects Hybrid

YASH MHATRE +91-7020257087 Roll No.: 241030901 mailto:ymmhatre_b23@ee.vjti.ac.in ELECTRICAL ENGINEERING Veermata Jijabai Technological Institute, Mumbai ï Yash Mhatre Education Veermata Jijabai Technological Institute (VJTI) • Bachelor of Technology in Electrical Engineering AUG.2025-CURRENT Veermata Jijabai Technological Institute (VJTI) CGPA: 8.61 • June 2021-May 2024 Diploma in Electrical Engineering Percentage:96.89 • M.G.V HIGHSCHOOL May 2020 SSC,Board Percentage: 91.80 Experience Brihanmumbai Electricity Supply and Transport Undertaking • (JUNE23-JULY23) Industrial Trainee Mumbai – learn about maintenance of underground transmission lines, & fault detection. – Gained knowledge of preventive maintenance and power distribution systems. The Tata Power Company Limited (JUNE25-JULY25) • INTERNSHIP Mumbai – Gained training of the cable department and different types of HT/EHV cables – Learned cable maintenance practices, fault locating, and jointing techniques. Personal Projects Hybrid Power Plant Analysis • – Analysed a village site for a solar-wind hybrid power plant. – Evaluated load requirements using electricity bills. – Calculated solar panel and wind turbine needs. – Tools & technologies used: MULTISIM IOT Based Electrical Machine Protection System • - This project is designed to continuously monitor the health of a three-phase induction motor using current, voltage, and temperature sensors connected to an ESP32 development board. – Tools & technologies used: Arduino IDE, Technical Skills - Languages: Basic C, Python, Basic MATLAB,Python(Libraries) - Software Tools: Multisim ,MATLAB ,Arduino IDE Positions of Responsibility Head Of Operation (Electrical • Engineering Student Association) Achievements VJTI - ALCHEMY 2025 Project competition March-2025 first price.